src/main/java/core/basesyntax/service/WriterService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface WriterService {
+    void writeToFile(String report, String path);
```

Comment:
```suggestion
    void writeToFile(String text, String path);
```
it can write any text I believe, not only report